test/submission.browser.js

```diff
@@ -125,6 +125,8 @@ test('Happy path', async t => {
     .expect(dashboard.statuses.textContent)
     // TODO this might cause a race condition
     .eql('Submitted')
+
+  await t.wait(5000)
```

Comment:
why?

Reply:
This is because the mecaExport is asynchronous so (annoyingly) there's no browser signal to indicate when they're done, so for now we're just waiting for some time to let them finish.

Reply:
ok, this needs a comment and an issue raised to fix it in the future.

Reply:
This is no longer necessary - the wait is now inside `navigationHelper.submit()`